pkg/api/registry/registry.proto

```diff
@@ -22,61 +22,41 @@ message Destination {
     uint32 weight = 2;
 }
 
-message NetworkServiceManager {
-    string name = 1;
-    string url = 2;
-    google.protobuf.Timestamp expiration_time = 3;
-    string state = 4;
-}
-
 message NetworkServiceEndpoint {
     string name = 1;
     string payload = 2;
     string network_service_name = 3;
-    string network_service_manager_name = 4;
-    map<string, string> labels = 5;
-    string state = 6;
-
+    map<string, string> labels = 4;
+    string state = 5;
```

Comment:
What are we doing with State?

Reply:
In monorepo this field used for keeping the state of NSE POD. Looks like we need to remove this field because we going to the independence of k8s